packages/medusa/src/api-v2/admin/stores/validators.ts

```diff
@@ -41,6 +47,10 @@ export class AdminPostStoresStoreReq {
   @IsString()
   name?: string
 
+  @IsOptional()
+  @IsArray()
+  supported_currency_codes?: string[]
```

Comment:
Just a nit, but I think we can drop the `supported_` prefix, discardable.

Reply:
@riqwan I was a bit torn TBH, and I decided to add it to distinguish between "all currency codes" and the ones that the store decided to support.